### PR TITLE
Vars

### PR DESCRIPTION
**Pull request by pixie79**

### CODE REVIEW

File: app/config/core.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
-  'keycloak_client_id': process.env.KEYCLOAK_CLIENT_ID || 'operational-data-api',
+  'logLevel': process.env.API_COP_LOG_LEVEL || 'info',
+  'dbProtocol': process.env.DB_COP_PROTOCOL || 'postgres://',
+  'dbPort': process.env.DB_COP_PORT || '5432',
+  'dbHostname': process.env.DB_COP_HOSTNAME || 'localhost',
+  'dbOptions': process.env.DB_COP_OPTIONS || '',
+  'dbName': process.env.DB_COP_OPERATION_DBNAME || 'operation',
+  'dbUsername': process.env.DB_COP_OPERATION_AUTHENTICATOR_USERNAME || 'authenticatoroperation',
+  'dbPassword': process.env.DB_COP_OPERATION_AUTHENTICATOR_PASSWORD || 'auth1234',
+  'dbConnectionString': process.env.DB_COP_CONNECTION_STRING || `${process.env.DB_COP_PROTOCOL}${process.env.DB_COP_OPERATION_AUTHENTICATOR_USERNAME}:${process.env.DB_COP_OPERATION_AUTHENTICATOR_PASSWORD}@${process.env.DB_COP_HOSTNAME}:${process.env.DB_COP_PORT}/${process.env.DB_COP_OPERATION_DBNAME}${process.env.DB_COP_OPTIONS}`,
```
Comment (p3dr0migue1) [inline app/config/core.js]: :bomb:

Comment (pixie79) [reply app/config/core.js]: :) updated

File: .drone.yml [hunk 2]
```diff
@@ -67,24 +92,36 @@ pipeline:
...
+      - export KEYCLOAK_PROTOCOL=$${STAGING_KEYCLOAK_PROTOCOL}
+      - export KEYCLOAK_REALM=$${STAGING_KEYCLOAK_REALM}
+      - export COP_DATA_API_KEYCLOAK_CLIENT_ID=$${STAGING_COP_DATA_API_KEYCLOAK_CLIENT_ID}
+      - export COP_DATA_API_LOG_LEVEL=$${STAGING_COP_DATA_API_LOG_LEVEL}
+      - export COP_DATA_API_DB_CONNECTION_STRING=$${STAGING_COP_DATA_API_DB_CONNECTION_STRING}
+      - export COP_DATA_API_PORT=$${STAGING_COP_DATA_API_PORT}
+      - export DB_DATA_COP_OPERATION_SCHEMA=$${STAGING_DB_DATA_COP_OPERATION_SCHEMA}
+      - export COP_DATA_API_IMAGE_REPO=$${STAGING_COP_DATA_API_IMAGE_REPO}
+      - export NGINX_PROXY_IMAGE_REPO=$${STAGING_NGINX_PROXY_IMAGE_REPO}
+      - export NGINX_PROXY_IMAGE_TAG=$${STAGING_NGINX_PROXY_IMAGE_TAG}
```
Comment (tomfitzherbert) [inline .drone.yml]: The names here don't match those on lines 107 & 108

Comment (pixie79) [reply .drone.yml]: yup staging is later

**Pull request merged**